docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
 
 :::
 
-For more information on the validity and availability process, be sure to visit the
-[parachain protocol](../learn/learn-parachains-protocol.md) page.
+Compared to synchronous backing, contextual execution shifts from being the parablock ancestors
+included in the relay chain to being the latest ancestor parablock pushed into the unincluded
+segment. This allows collators to build parablocks earlier, giving them plenty of time to fit more
+transactions and prepare block candidates for backing and inclusion.
+
+### Asynchronous Backing Mechanics
+
+In asynchronous backing, parablock generation is decoupled from backing and inclusion. Compared to
+[synchronous backing](#synchronous-backing-mechanics):
+
+- Parablocks are not rushing to being backed, they are not placed on the conveyor belt when they are
+  generated
+- Filled blocks are placed in the unincluded segment, and from there, they are injected onto the
+  conveyor belt
+- The conveyor belt can carry multiple parablocks ([pipelining](#pipelining)), and a parablock can
+  be in a different stage from another one being built if it abides by the parameters set forth by
+  the asynchronous backing configuration.
+- Backing and inclusion events can happen within the same relay chain block (i.e. in 6 seconds)
+- The last parablock in the unincluded segment is used as execution context to generate a new
+  parablock and, because the unincluded segment can carry multiple parablocks, parachains can take
+  more time to fill new blocks while the segment is emptied.
+
+The diagram below shows parablocks on their way from being generated to being backed and included
+into the relay chain in the context of asynchronous backing.
+
+![async-backing](../assets/async-backing.png)
+![async-backing-legend](../assets/async-backing-legend.png)
+
+Parablock 1 (P1) is included, and P2 is backed within R1. In the meantime, the unincluded segment is
+full and contains P3-5. Because P6 is not rushing to be backed, it can be filled in 6s while P2 is
+included and P3 is backed in R2. This is possible because the context execution shifted from being
+the latest included parablock to being the latest ancestor in the unincluded segment (i.e. P5).
+
+In this scenario, blockspace demand decreases and as a consequence P6 is filled only 50% using 6
+seconds. Next block P7 can be filled in 12 seconds while the unincluded segment is emptied. While
+filling up P7, P3 and P4 are included in R3 and R4, respectively. In 24 seconds, four parablocks, P1
+to P4, have been included in the relay chain, P5 has been backed, and two more blocks have been
+filled and pushed to the unincluded segment, ready to be backed. The unincluded segment is now 2/3
+filled.
+
+Collators were able to make better use of blockspace, including 2x more data, by doubling block
+generation time from 6 to 12 seconds in a period of lower demand. This was possible due to the
```

Comment:
We previously only allowed 0.5 seconds of block generation time, since we needed the other 5.5 seconds to get the block backed on chain.